pyccl/__init__.py

```diff
@@ -1,3 +1,7 @@
+# Sets the environment variable for default config path if it does not exist yet
+from os import environ, path
+if environ.get("CCL_PARAM_FILE") is None:
+    environ["CCL_PARAM_FILE"] = path.dirname(path.abspath(__file__)) + '/ccl_params.ini'
```

Comment:
Just checking -- does the C code also have an environment variable check like this?

Reply:
Well, the C code will first try the environment, then look for the hard coded install_dir/ccl_params.ini, then crash if not found. If the environment variable is defined by the user then both C and python codes will use the same config file